docs/getting-started/README.md

```diff
@@ -40,7 +40,7 @@ your cluster, you will need this for your Webhook in later steps.
 - Create the _getting-started_ namespace, where all our resources will live.
   - `kubectl create namespace getting-started`
 - [Create the admin user, role and rolebinding](./rbac/admin-role.yaml)
-  - `kubectl apply -f ./docs/getting-started/rbac/admin-role.yaml`
+  - `kubectl -n getting-started apply -f ./docs/getting-started/rbac/admin-role.yaml`
```

Comment:
Can we set the default either consistently inside the file or when applying? Currently roughly half the files have the namespace hardcoded while the other half is applied like this.

Reply:
done.